org.gustavoperedo.VideoDownloader.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id" : "org.gustavoperedo.VideoDownloader",
```

Comment:
Do you own this domain? If not, it should be `com.github.GustavoPeredo.VideoDownloader`.

Reply:
I changed it to io.github.gustavoperedo.VideoDownloader (following the [documentation](https://docs.flatpak.org/en/latest/conventions.html#application-ids).)

I have another app that I used the org.gustavoperedo prefix, for me to update it, do I just have to change the manifest's app-id? (The app is already on flathub)

Reply:
No, it's fine for existing app to stay that way. We try to limit friction caused by ID changes to minimum.